designer/server/src/main/scala/pl/touk/nussknacker/ui/api/StatisticsApiHttpService.scala

```diff
@@ -35,7 +39,14 @@ class StatisticsApiHttpService(
   expose {
     endpoints.registerStatisticsEndpoint
       .serverSecurityLogic(authorizeKnownUser[Unit])
-      .serverLogicSuccess { _ => _ =>
+      .serverLogicSuccess { _ => request =>
+        // todo change to groupMapReduce in scala 2.13
+        val groupedByName = request.statistics
+          .groupBy(_.name.entryName)
+          .map { case (k, v) =>
+            k -> v.size.toLong
+          }
+        db.write(groupedByName)
```

Comment:
we "swallow" the potential exception here. Maybe it would be nice to log it? 
BTW shouldn't we create a private method for this logic?